pandas/core/generic.py

```diff
@@ -3345,37 +3349,85 @@ def to_latex(
             ``\begin{{}}`` in the output.
 
             .. versionadded:: 1.2.0
-        {returns}
+
+        Returns
+        -------
+        str or None
+            If buf is None, returns the result as a string. Otherwise returns None.
+
         See Also
         --------
         Styler.to_latex : Render a DataFrame to LaTeX with conditional formatting.
         DataFrame.to_string : Render a DataFrame to a console-friendly
             tabular output.
         DataFrame.to_html : Render a DataFrame as an HTML table.
 
+        Notes
+        -----
+
+        .. note::
+           As of v1.5.0 this method has changed to use the ``Styler`` implementation of
+           ``to_latex`` and no longer uses the DataFrameRenderer. It is advised that
```

Comment:
``DataFrameRenderer`` is an internal class.
I suggest against mentioning it in the docstring.

Reply:
fair point, will readdress